### PR TITLE
fix python crash with NEURON8.1.0

### PR DESCRIPTION
**Pull request by arnaudon**
this is fix related to CELLS-42

### CODE REVIEW

**Comment by wvangeit** [comment]
Did you test this with older versions of Neuron?

File: bluepyopt/ephys/simulators.py [hunk 1]
```diff
@@ -121,6 +121,8 @@ def run(
         """Run protocol"""
 
         self.neuron.h.tstop = tstop
+        self._pc = self.neuron.h.ParallelContext()
+        self._pc.mpiabort_on_error(0)
```
[the remark this answers is not in the document]
Comment (nrnhines) [reply bluepyopt/ephys/simulators.py]: It should do the right thing if there is no MPI support. Ie. no abort. Abort can only happen if pc.nhost() > 1 and (default)
pc.mpiabort_on_error(1)

**Comment by arnaudon** [comment]
I close this, this is not solving it, it's an issue with ipyparallel, with dask it works without this. Sorry for the noise here.

**Comment by mgeplf** [comment]
> it's an issue with ipyparallel

What's the issue w/ ipyparallel?

**Comment by DrTaDa** [comment]
@mgeplf when using ipyparallel, when NEURON crashes, it also crashed the whole python script (core dump). The issue is being investigated in the ticket CELLS-42

**Comment by mgeplf** [comment]
Thanks; I'll follow along there.

**Comment by arnaudon** [comment]
In fact, this is needed with for example dask-mpi parallelization to prevent neuron to send kill signal and crash everything.

**Pull request closed without merging**